jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
   comment:
-    - add: 
+    - add:
         body: |~
           {{ or .overrides.comment "" | indent 10 }}
+{{end}}
+{{ if .overrides.label }}
```

Comment:
I think this is probably similar to comment, someone can use `jira edit -o label=foobar` but maybe they should see a labels section if no overrides are present?  So I would expect this to look something like:
```
labels:{{range label in .fields.labels}
  - add: {{label}}{{end}}
  - add: {{ or .overrides.label "" }}
```

(I am assuming lables pre-exist in an issue, so perhaps we show users current labels and allow them to add more?)